docs/tutorial/from-zero-to-hero-write-your-first-kubernetes-charm/create-a-minimal-kubernetes-charm.md

```diff
@@ -15,7 +15,7 @@ If you are familiar with  Juju, as we assume here, you'll know that, to start us
 
 As you already know from your knowledge of Juju, when you deploy a Kubernetes charm, the following things happen:
 
-1. The Juju controller provisions a pod with two containers, one for the Juju unit agent and the charm itself and one container for each application workload container that is specified in the `containers` field of a file in the charm that is called `charmcraft.yaml`. 
+1. The Juju controller provisions a pod with at least two containers, one for the Juju unit agent and the charm itself and one container for each application workload container that is specified in the `containers` field of a file in the charm that is called `charmcraft.yaml`. 
```

Comment:
Sadly that's not quite precise... though maybe we shouldn't overload new charmers with too much info either?

For a "normal" charm, the pod contains 3 containers:
- startup
- charm
- workload

However, the workload container is optional, a charm without containers is totally valid... so it could be two containers, just not in a way described here.

Maybe we could add qualifier, e.g. in the sentence above?
`when you deploy a (typical|simple|standard) Kubernetes charm`

Reply:
I made a mistake doing small adjustments in this PR instead of sticking to removing the chapters. I agree with making an improvement like that, but I think I need to avoid making more text improvements in this PR otherwise it's never going to get merged.

Reply:
Although in this case I did perhaps make it worse. I've removed the number entirely. Mentioning the startup container seems maybe too much knowledge right now? I think we can fix that in a different PR if we want to.